.github/workflows/scan.yaml

```diff
@@ -0,0 +1,144 @@
+name: Trivy Scan
+on:
+  workflow_dispatch: {}
+  schedule:
+    # run every day at 3:07am UTC
+    - cron: '7 3 * * *'
+
+env:
+  GOMPLATE_VERSION: v3.11.4
```

Comment:
What's Gomplate? Do we need this?

Reply:
that was a leftover of a previous attempt at generating some summary from trivy's output, removed 👍